scheduler/src/cook/rest/api.clj

```diff
@@ -1968,10 +1969,14 @@
   ::jobs and ::groups, which specify the jobs and job groups."
   [conn {:keys [::groups ::job-pool-name-maps] :as ctx}]
   (try
-    (log/info "Submitting jobs through raw api:"
-              (map (fn [job-pool-name-map]
-                     (update job-pool-name-map :job #(dissoc % :command)))
-                   job-pool-name-maps))
+    (let [jobs (map (fn [job-pool-name-map]
+                      (update job-pool-name-map :job #(dissoc % :command)))
+                    job-pool-name-maps)]
+      (log/info "Submitting jobs through raw api:" jobs)
+      (doseq [{{:keys [uuid, user]} :job} jobs]
+        (passport/log-passport-event {:job-uuid (str uuid)
+                                      :user user
+                                      :event-type passport/api-job-submission})))
```

Comment:
Would it make more sense to log this event write after the `d/transact` call returns lower in this function? That's when we're doing similar things like incrementing the queue length and marking the `jobs-created` metric.

Reply:
I view "job submission" as the point at which we have the jobs and are beginning to create them, not after we create them (i.e. after the `d/transact` call returns), so to me I think that the beginning of the function is the most logical point to log the event. 

Also, if the `d/transact` function begins to slow down for some reason, we want the passport events to show that behavior. Specifically, we would want the passport metrics aggregator to show that the time between `api-job-submission` and, for example, `pod-launching` has increased. If we log `api-job-submission` after the `d/transact` call, that difference would not be captured.

Reply:
SGTM.

In that case, is there a way we can avoid the extra `let` and the new `jobs` binding (there is already a `jobs` binding below)? Maybe leave the "Submitting jobs through raw api" code as it was and move the passport event logging into the existing `let`, right below the existing `jobs` binding?

Nitpick: lexicographic map keys for the passport event map

Reply:
If we moved it into the second let, the passport logging would have to go below the bindings, right? Similar to my other comment, I'd worry that if a function call like (d/db conn) becomes slow, we wouldn't capture that in the Passport event logs. I also sort of think that organizationally I prefer seeing the two logging statements within the same let function & occurring immediately in the `create-jobs!` function. 

But it definitely isn't strictly necessary to have it in the first `let` statement, and if you feel that it's most beneficial to avoid the extra `let` I can do that in the next revision. Let me know!

Reply:
Sounds reasonable, I made a suggestion below that keeps them next to each other but removes the need for an additional `let`